src/utils.js

```diff
@@ -1,12 +1,10 @@
-/*import * as fs from "fs";
+import * as fs from "fs";
 
 const writeJSONFile = (jsonObj) => {
   var file = fs.createWriteStream("tree.json");
-  file.on("error", function (err) {
-  });
+  file.on("error", function (err) {});
```

Comment:
can you add some error reporting here, in the error handling function? it will probably be useful when the program fails (e.g. cannot write because of permissions).

Reply:
done